src/App.jsx

```diff
@@ -14,57 +15,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selected, setSelected] = useState('Jam');
+  const clearCart = () => {
+    setSelected('');
+  };
```

Comment:
The `clearCart` function can be renamed to `clearSelection` because it clears the selection, not the cart.